docs/aerodromes/classc/Melbourne.md

```diff
@@ -132,6 +128,23 @@ Remember to pass traffic information to both aircraft.
     **ML ADC:** "JST515, traffic is a helicopter, 2nm northwest of the field, tracking for Essendon and maintaining own separation with you, runway 16, cleared to land"  
     **JST515:** "Runway 16, cleared to land, JST515"
 
+## Workload Management
+During busy events, such as [Milk Run Monday](../../../controller-skills/milkrun), the **SMC** controller may end up with a much higher workload than the **ACD** controller. Additionally, delays may need to be implemented for aircraft requesting pushback, so as to not overload the taxiways and holding points.
+
+### Pushback Requests on ACD
+To mitigate this, pushback requests may be done on **ACD** frequency, to balance the workload. A few steps must be followed to properly execute this procedure.
+
+1. **SMC** and **ACD** coordinate to implement the procedure, due to high **SMC** workload.
+2. **SMC** coordinates with **ADC** in order to have the [ATIS](#acd-pushback-requests) updated.
+3. When **ACD** has finished issuing an airways clearance, they will **remind** pilots to *"Contact me when ready for pushback"*.
```

Comment:
Can we add a line that ACD should cross check any applicable slot (COBT) times for the aircraft in this process?

Reply:
Yeah was thinking about that. How would that look? How is the current COBT data presented and whereabouts can the controller find it?

Reply:
Perhaps just a general comment like Matt suggested above is suitable here, in case we end up with a different slot system in the future. This procedure is more about the high level steps which should be taken